### PR TITLE
Fixed spelling mistakes in documentation and variable names

### PR DESCRIPTION
**Pull request by michael-baker**
Fixed up some spelling mistakes in variable names and some unit test methods.

### CODE REVIEW

**Comment by MichaCo** [comment]
Nice job, thank you very much!

File: src/CacheManager.Core/Internal/BackplaneMessage.cs [hunk 1]
```diff
@@ -99,23 +99,23 @@ public static BackplaneMessage Deserialize(string message)
 
             var tokens = message.Split(new[] { ":" }, StringSplitOptions.RemoveEmptyEntries);
 
-            var ident = tokens[0];
+            var indent = tokens[0];
```
Comment (MichaCo) [inline src/CacheManager.Core/Internal/BackplaneMessage.cs]: not sure why you changed that one. `ident` was short for identity ;)

Comment (michael-baker) [inline src/CacheManager.Core/Internal/BackplaneMessage.cs]: Oops, I'll back that out.

**Comment by michael-baker** [comment]
Fixed the over zealous fixing. Hope it's OK now.

**Comment by MichaCo** [comment]
Great, thank you :+1:

**Pull request merged**